pre-build-check.sh

```diff
@@ -1,12 +1,5 @@
 #!/usr/bin/env bash
 
-# Check 1: UHN VPN is up
```

Comment:
Seems like it'd be easier to take the clause looking for $CANDIG_HOST out of the if statement below, rather than take away the calculation?

Reply:
removed the if statement too

Reply:
leave in the $SILENT_MODE check...that is still useful.

Reply:
I don't get what you mean, the silent mode is just to check whether to print the message about the uhn vpn or not, but we aren't doing that anymore? Do you mean you want to do a silent mode check for the domain check too?

Reply:
I don't know; you're probably right. I think that my impulse is usually to remove as little code as possible, but perhaps that isn't very helpful. I'll go ahead and approve it.